content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md

```diff
@@ -1,38 +1,37 @@
 ---
-title: "Shutting Down and Restart Cluster Gracefully"
-description: "Demonstrate how to shut down and restart kubernetes cluster Gracefully"
+title: "Cluster Shutdown and Restart"
```

Comment:
Is it more exact to update the title to `shutting down and restart all nodes in the cluster`?

Reply:
![image](https://user-images.githubusercontent.com/65327072/96941655-95f7d900-1505-11eb-8406-7ed996dec2c6.png)
[openshift doc](https://docs.openshift.com/container-platform/4.5/backup_and_restore/graceful-cluster-shutdown.html)